### DISCUSSION TITLE
Adding Custom "Off-Type" Icons for State Representation in the Dashboard

### DISCUSSION THREAD

**Discussion by Menelao147** [elax46/custom-brand-icons#785] [Q&A] [answered]
Hi everyone,
In my current dashboard, I am using three different icons to represent the various states of my car:

`mdi:car-off` (car not in garage)
`mdi:car` (car in garage)
`mdi:car-electric` (car connected and charging)

I would like to replace these with a more specific set of icons using `hi:tesla-car`.
For consistency, I also created `phi:tesla-car-off` and `phi:tesla-car-electric`.
However, when I submitted a pull request for these, it was closed because: _“Off-type icons will not be accepted, as Home Assistant makes it visible when an entity is on or off, and it is useless to weigh down the library.”_

Given my need to represent multiple distinct states, would it be possible to add an "off" variant in this case? I’d like to understand if there is an exception or recommended approach for cases like this, where the icon set is meant to indicate more than just binary on/off states.

Thanks!

**Comment by elax46** [comment]
I think @rchiileea  can give you a proper answer to the off icons question, so I'll leave it up to him to answer!

**Comment by rchiileea** [comment, marked as answer]
of type icons end up weighing down the libary and HA will show on and off by icon color change, you could have the icon as normal for in garage, red for out of garage and blue for charge.
iicon section of custom button cardin custom button card
```
      styles:
          icon: |
                [[[ 
                   if ( device_tracker.ioniq_5_location.state >= home) return 'white';
                   else if (device_tracker.ioniq_5_location.state >= away) return 'red';
                   else if (binary_sensor.ioniq_5_ev_battery_plug >= plugged) return 'blue';
                   else if (sensor.ioniq_5_ev_battery_level >= 100) return 'green';
                   else return 'yellow';
                ]]]
```

one icon multiple functions
